src/backend/mail.py

```diff
@@ -1,71 +1,16 @@
-                        },
-                        "Text": {
-                            "Charset": "UTF-8",
-                            "Data": render_to_string(template_name + ".txt", template_details),
-                        },
-                    },
-                    "Subject": {"Charset": "UTF-8", "Data": subject_line},
-                },
-                Source=f"{settings.MAIL['SEND_NAME']} <{settings.MAIL['SEND_ADDRESS']}>",
-            )
-        elif settings.MAIL["SEND_MODE"] == "SENDGRID":  # pragma: no cover
-            data = {
-                "personalizations": [{"to": [{"email": send_to}], "subject": subject_line}],
-                "from": {"email": settings.MAIL["SEND_ADDRESS"], "name": settings.MAIL["SEND_NAME"]},
-                "content": [
-                    {"type": "text/plain", "value": render_to_string(template_name + ".txt", template_details)},
-                    {"type": "text/html", "value": render_to_string(template_name + ".html", template_details)},
-                ],
-            }
-            sg.client.mail.send.post(request_body=data)
-        elif settings.MAIL["SEND_MODE"] == "SMTP":  # pragma: no cover
-            import smtplib
-            from email.mime.multipart import MIMEMultipart
-            from email.mime.text import MIMEText
-
-            if settings.MAIL["SMTP_USE_SSL"]:
-                smtp = smtplib.SMTP_SSL(settings.MAIL["SEND_SERVER"])
-            else:
-                smtp = smtplib.SMTP(settings.MAIL["SEND_SERVER"])
-            smtp.connect(settings.MAIL["SEND_SERVER"])
-            smtp.set_debuglevel(False)
-            smtp.login(settings.MAIL["SEND_USERNAME"], settings.MAIL["SEND_PASSWORD"])
-            sender = f"{settings.MAIL['SEND_NAME']} <{settings.MAIL['SEND_ADDRESS']}>"
-
-            data = MIMEMultipart("alternative")
-            data["To"] = send_to
-            data["From"] = sender
-            data["Subject"] = subject_line
-
-            data.attach(MIMEText(render_to_string(template_name + ".txt", template_details), "plain"))
-            data.attach(MIMEText(render_to_string(template_name + ".html", template_details), "html"))
-
-            smtp.sendmail(sender, send_to, data.as_string())
-    else:
-        print(
-            f"Sending email '{subject_line}' to {send_to} using template {template_name} with details {template_details}"
-        )
+from django.core.mail import EmailMultiAlternatives
+from anymail.message import attach_inline_image_file
+from django.conf import settings
+from os import path
+
+
+def send_email(send_to: str, subject_line: str, template_name: str, **template_details) -> None:
+    """Wrapper function around `django.core.mail.send_mail` to simplify use of templates."""
+    msg = EmailMultiAlternatives(
+        subject_line, render_to_string(template_name + ".txt", template_details), None, [send_to]
+    )
+    cid = attach_inline_image_file(msg, path.join(settings.BASE_DIR, "logo.png"))
```

Comment:
Does this work? The docs seem to suggest that their [convenience methods only work on subclasses of their own message mixin](https://anymail.readthedocs.io/en/stable/sending/anymail_additions/).

Reply:
Seems to. I was able to send an email with this image through both Anymail ESPs and normal Django SMTP